pkgs/by-name/bs/bsync/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+}:
+
+stdenv.mkDerivation {
+  pname = "bsync";
+  version = "20231221-25f7773";
```

I think unstable versions are usually done in this format. You'll have to redo the commit message as well

```suggestion
  version = "unstable-2023-12-21";
```